support-frontend/assets/components/subscriptionCheckouts/address/addressFields.tsx

```diff
@@ -77,7 +77,7 @@ function statesForCountry(country: Option<IsoCountry>): React.ReactNode {
 
 export function AddressFields({ scope, ...props }: PropTypes): JSX.Element {
 	return (
-		<div>
+		<div data-component={`${scope}AddressFields`}>
```

Comment:
Is this linked to the `data-link-name` property in the <Select?? Or is this for readability in the inspector?

Reply:
`data-component` helps to differentiate or "contain" events. The in multiple containers. Probably not massively useful for this, but it felt like good practise to define a boundry.